api/docs/http_catalog.md

```diff
@@ -8,7 +8,7 @@ Find
 
 Retrieve a catalog.
 
-    POST /puppet/v3/catalog/:nodename?environment=:environment
+    POST /puppet/v3/catalog/:nodename
```

Comment:
Wait, sorry, I see you're treating environment as part of the POST data instead. If that's the case, though, I'd like to see environment added to the parameters section, to make sure it's not overlooked in a POST request. (Catalog is the only traditional endpoint we POST to, right?)